fast_agave/filters.py

```
@@ -1,10 +1,8 @@
-from typing import List
-
 from cuenca_validations.types import QueryParams
 from mongoengine import Q
 
 
-def generic_query(query: QueryParams, excluded: List[str] = []) -> Q:
+def generic_query(query: QueryParams, excluded: list[str] = []) -> Q:
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)